packages/server/src/index.ts

```diff
@@ -75,7 +75,9 @@ passport.deserializeUser<string>(function (id, callback) {
       if (user) {
         callback(null, user);
       } else {
-        callback(new Error(`No user with ID: ${id}`));
+        // This may be the case if a user has been deleted, but the user session persists.
+        // Sending null as the second argument invalidates the session.
+        callback(null, null);
```

Comment:
Without this change, the deleted user essentially gets "blacklisted" from the site: they just get the response body "No user with ID: <id>", until they clear their cache, which is not obvious to laypeople.

See also: https://github.com/jaredhanson/passport/issues/6